ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/forkchoice/ForkChoiceUpdateData.java

```diff
@@ -61,6 +61,8 @@ public ForkChoiceUpdateData(
           new ForkChoiceState(
               forkChoiceState.getHeadBlockRoot(),
               forkChoiceState.getHeadBlockSlot(),
+              // no block number for the terminal block
```

Comment:
Maybe something clearer like `We don't have data for terminal block number`. Just nit